README.md

```diff
@@ -1,3 +1,6 @@
+<p align="center">
+    <img width="150px" src="https://cdn.rawgit.com/MichaelMure/git-bug/logo/misc/logo/logo-alpha-flat-bg.svg">
```

Comment:
Rawgit doesn't support a branch argument and simply linking the file from the repo will not work because github doesn't set the correct content type when serving the file so it will not appear properly in the readme.

Reply:
Seems to work for me
https://github.com/vandesign/git-bug/blob/git-bug-logo/README.md

Reply:
Ok, now I'm confused. Well, either way, it works now.

Reply:
I'm confused too. You're referencing the logo branch in this url:
[https://cdn.rawgit.com/MichaelMure/git-bug/**logo**/misc/logo/logo-alpha-flat-bg.svg](https://cdn.rawgit.com/MichaelMure/git-bug/logo/misc/logo/logo-alpha-flat-bg.svg)
Will it work when you remove the branch?

Reply:
Ho my bad, I read too fast and thought it was the logo folder. I'll change it to master.